app/assets/javascripts/admin/all.js

```diff
@@ -50,6 +50,8 @@
 //= require textAngular.min.js
 //= require i18n/translations
 //= require darkswarm/i18n.translate.js
-
+//= require moment
+//= require moment/es.js
+//= require moment/fr.js
```

Comment:
Doesn't `require_tree` do the trick? If I'm not mistaken (I'm not very familiar with the Rails asset pipeline), it'd require all the files in `moment/`